web/src/timerender.ts

```diff
@@ -308,12 +324,30 @@ export function initialize(): void {
     last_update = startOfToday();
 }
 
+function is_topic_timestamp_entry(entry: UpdateEntry): entry is UpdateEntryWithKey {
+    return "conversation_key" in entry;
+}
```

Comment:
I think it might be nicer to just not have the`is_topic_timestamp_entry`  function and instead have `maybe_add_update_list_entry` just be passed which list to add things to as a parameter (or have two copies); this construction makes we worry that we're going to accidentally mix up the two data structures somehow.

Reply:
Passing a reference to the list is a good idea. I prefer this approach over having two copies of `maybe_add_update_list_entry`.